lib/ref_arch_setup/url_helper.rb

```diff
@@ -0,0 +1,88 @@
+# General namespace for RAS
+module RefArchSetup
+  # Code to illustrate a Beaker-based method of deriving a tarball URL
+  class UrlHelper
+    # Determine is a given URL is accessible
```

Comment:
is --> if

Reply:
This was directly copied in from Beaker (`web_helpers.rb`) to complete the example; I'll add a note to fix it there as part of the update.